apps/web/src/components/editor/index.tsx

```diff
@@ -121,6 +121,23 @@ export default function TabsView() {
   const isTOCVisible = useEditorStore((store) => store.isTOCVisible);
   const [dropRef, overlayRef] = useDragOverlay();
 
+  useEffect(() => {
+    const onKeyDown = (event: KeyboardEvent) => {
+      if (event.ctrlKey && event.altKey && event.key === "ArrowRight") {
+        event.preventDefault();
+        useEditorStore.getState().openNextSession();
+      }
+      if (event.ctrlKey && event.altKey && event.key === "ArrowLeft") {
+        event.preventDefault();
+        useEditorStore.getState().openPreviousSession();
+      }
+    };
+    document.body.addEventListener("keydown", onKeyDown);
+    return () => {
+      document.body.removeEventListener("keydown", onKeyDown);
+    };
+  }, []);
```

Comment:
I've moved the `onKeyDown` handler for this keyboard shortcut here as I noticed that the `readonly` notes weren't handled.

Reply:
Makes sense.